### PR TITLE
Feat : Added Homepage Styles

### PR DESCRIPTION
**Pull request by srejitk**
This Feature Request adds the following page styles :

1. Home Page

Home

### CODE REVIEW

File: Stylesheets/Homepage/Homepage.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    overflow-x: scroll;
+}
+
+.glass__categories {
+    grid-area: categories;
+    width: 100%;
+    background-color: var(--forground-color);
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
```
Comment (nakulsharma15) [inline Stylesheets/Homepage/Homepage.css:18]: You can extract this in a utility class

**Review by nakulsharma15 — APPROVED** [review]
Just gave a minor suggestion. Everything else seems good

**Review by krishanu2911 — APPROVED** [review]
No changes required well-written code

File: Stylesheets/Homepage/Homepage.css (hunk 1 above)
Comment (srejitk) [reply Stylesheets/Homepage/Homepage.css:18]: Thank You. Will refactor this.

**Pull request merged**